tests/mock_network/churn.rs

```diff
@@ -18,7 +18,254 @@ use routing::{
     Authority, Event, EventStream, NetworkConfig, NetworkParams, Prefix, XorName,
     QUORUM_DENOMINATOR, QUORUM_NUMERATOR,
 };
-use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet};
+use std::{
+    collections::{BTreeMap, BTreeSet, HashMap, HashSet},
+    usize,
+};
+
+#[test]
+fn aggressive_churn() {
```

Comment:
Wondering if it would make sense to have a test not sending any messages. for another Pr though.

Reply:
Sure, the idea is that we can now have all kinds of tests by just using different parameters. Not sending any messages isn't currently supported though, but should be easy to add.